modules/virt-specializing-windows-sysprep.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-automating-windows-sysprep.adoc
+
+[id="virt-specializing-windows-sysprep_{context}"]
+= Specializing a Windows VM
+
+Specializing a virtual machine configures the computer-specific information from the image onto the VM.
+
+[IMPORTANT]
+====
+You must generalize the root disk prior to specializing the virtual machine.
+====
+
+.Procedure
+
+. Use the {VirtProductName} web console VM wizard to xref:../../virt/virtual_machines/virt-create-vms.adoc#virt-create-vms[create a new Windows VM]
```

Comment:
why we need brackets around VirtProductName

Reply:
Please review the resolved conversations and the preview link of the content. This is AsciiDoc formatting.